src/test/java/org/jfrog/hudson/pipeline/integration/PipelineTestBase.java

```diff
@@ -214,6 +228,7 @@ static String getProjectPath(String projectName) {
      */
     WorkflowRun runPipeline(String name) throws Exception {
         WorkflowJob project = jenkins.createProject(WorkflowJob.class);
+        Utils.injectStepIdParameter(project, "{\"stepId\":\"5\"}");
```

Comment:
Since the random reader may not be aware of the JFrog Pipelines integration here, let's add a description explaining what this line does.